### PR TITLE
Add two examples of how to run the code

### PR DESCRIPTION
**Pull request by shoyer**
I also uploaded some of the training data to a public cloud storage bucket.

I probably should have done this a long time ago...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +30,39 @@ From the source directory, execute each test file:
...
+Training data was created with `create_training_data.py`, but can also be
+downloaded from Google Cloud Storage:
+
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/burgers.h5
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/kdv.h5
+- https://storage.googleapis.com/data-driven-discretization-public/training-data/ks.h5
+
+We have two notebooks showing how to train and run parts of our model:
+- [Super resolution of Burgers' equation](https://colab.research.google.com/github/google/data-driven-discretization-1d/blob/master/notebooks/burgers-super-resolution.ipynb)
+- [Time integration of Burgers' equation](https://colab.research.google.com/github/google/data-driven-discretization-1d/blob/master/notebooks/time-integration.ipynb)
```
Comment (yohai) [inline README.md]: I think the time integration notebook is missing

Comment (shoyer) [reply README.md]: Good catch, fixed.

**Comment by shoyer** [comment]
@yohai I agree, but I've gotten a few emails asking me how to actually run this code. This is better than writing individual responses!

**Comment by shoyer** [comment]
Thanks for the review! I added some notes about how the notebooks are intended to be run in Colab.

**Pull request merged**